### ISSUE TITLE
Support output predict

### ISSUE THREAD

**Issue by thachkysanh1996** [A-bone1/Attention-ocr-Chinese-Version#47]
After trained model with your tutorial. (Using your dic.txt and fsns in data/train/tfexample.record). Then i use the model-checkpoint to infer 2 image. But the output is not right. Please help me! 

![screenshot from 2019-01-08 14-56-51](https://user-images.githubusercontent.com/35448859/50817381-35120780-1356-11e9-988d-36e85b9fa2c3.png)

**Comment by A-bone1** [comment]
@thachkysanh1996 This should be a problem with coding and decoding. You can check the relevant information. You can also use python2 to try the correctness of the result through this command 
"print( "b********".decode('utf-8'))""

**Comment by thachkysanh1996** [comment]
@A-bone1  thanks you !!!

**Comment by thachkysanh1996** [comment]
@A-bone1 Does your project work on photo ocr ? or just a image cropped with text

**Comment by A-bone1** [comment]
@thachkysanh1996  The type of task depends on your training data. If your training data is photo ocr, the model can work on photo ocr.

**Comment by thachkysanh1996** [comment]
@A-bone1 Thanks! I have another question. If training data is photo ocr with multiple text in image. How can i format the label ?

**Comment by A-bone1** [comment]
@thachkysanh1996 I have not tried this situation, but I think it is possible to add a text detection network in front of the text recognition network. It may be better to crop the detected text from the original image and then perform text recognition separately.

**Comment by thachkysanh1996** [comment]
@A-bone1  Thanks you !!!

**Comment by thachkysanh1996** [comment]
@A-bone1  I wrote the code to generate syntext data for fsns (Background white and put text on it). After training model 120k step, it will recognize the data like i syn. 

![ask](https://user-images.githubusercontent.com/35448859/51157063-caf6e680-18b0-11e9-9dc4-14c4454e9156.png)
![asdasda](https://user-images.githubusercontent.com/35448859/51157081-dfd37a00-18b0-11e9-82ea-a9f15c639b44.png)

But when test with cut image (in real). It cannot recognize and accuracy is very poor.
How can i make my model good prediction in actual image? How can you generate synthtext image for training? Please help me!

**Comment by A-bone1** [comment]
@thachkysanh1996 Is the prediction result of the test set similar to the training set accurate? If the training set differs greatly from the test set style, the accuracy is really poor. I think the generated training set should be as similar as possible to the test set. In addition, the cut image should not be distorted after reshaping.

**Comment by thachkysanh1996** [comment]
@A-bone1  Yes, the test set is text image from the real world. (Different with training set which i generated like image above). The accuracy is really poor. I think that my generate is very simple, not  inconsistent light condition like text image in the wild. 

In addition, the cut image like this. How can i know about distortion after the model reshape image? 

![fsns_train_08](https://user-images.githubusercontent.com/35448859/51160816-588f0200-18c2-11e9-9df1-e1f915823ee7.jpg)

**Comment by tuongtranngoc** [comment]
hi ! @thachkysanh1996  how to predict a single image (ramdom size)?? what file do you run?
